src/shave.js

```diff
@@ -55,11 +55,13 @@ export default function shave(target, maxHeight, opts) {
   }
 }
 
-const plugin = window.$ || window.jQuery || window.Zepto;
-if (plugin) {
-  plugin.fn.extend({
-    shave: function shaveFunc(maxHeight, opts) {
-      return shave(this, maxHeight, opts);
-    },
-  });
+if (typeof window !== 'undefined') {
```

Comment:
@nbryant first, thanks for the well thought out PR! The only thing I think we might be able to improve is the nested if check. 

I thought of this:
```
const plugin = typeof window !== 'undefined' ? window.$ || window.jQuery || window.Zepto : false;
```
So there would be only 1 if check ...

But the line is pretty long & sometime ternaries look confusing to people (especially when the line is long). 

What are your thoughts?